// @vkid/sdk/dist-sdk/cjs/index.js
'use strict';

var auth = require('./auth/auth.js');
var config = require('./core/config/config.js');
var widget = require('./core/widget/widget.js');
var events = require('./core/widget/events.js');
var types = require('./types.js');
var oneTap = require('./widgets/oneTap/oneTap.js');
var floatingOneTap = require('./widgets/floatingOneTap/floatingOneTap.js');
var types$1 = require('./widgets/floatingOneTap/types.js');
var oauthList = require('./widgets/oauthList/oauthList.js');
var types$2 = require('./widgets/oauthList/types.js');

var globalConfig = new config.Config();
/** Export Auth */ auth.Auth.__config = globalConfig;
var globalAuth = new auth.Auth();
/** Export Core Widget */ widget.Widget.__config = globalConfig;
widget.Widget.__auth = globalAuth;

Object.defineProperty(exports, 'WidgetEvents', {
	enumerable: true,
	get: function () { return events.WidgetEvents; }
});
Object.defineProperty(exports, 'Languages', {
	enumerable: true,
	get: function () { return types.Languages; }
});
Object.defineProperty(exports, 'Scheme', {
	enumerable: true,
	get: function () { return types.Scheme; }
});
exports.OneTap = oneTap.OneTap;
exports.FloatingOneTap = floatingOneTap.FloatingOneTap;
Object.defineProperty(exports, 'FloatingOneTapContentId', {
	enumerable: true,
	get: function () { return types$1.FloatingOneTapContentId; }
});
exports.OAuthList = oauthList.OAuthList;
Object.defineProperty(exports, 'OAuthName', {
	enumerable: true,
	get: function () { return types$2.OAuthName; }
});
exports.Auth = globalAuth;
exports.Config = globalConfig;
